test/test_autograd.py

```diff
@@ -6895,6 +6856,49 @@ def fn(v):
             nnz = 0 if empty_nnz else 5
             _test(sparse_size + dense_size, len(sparse_size), nnz, device)
 
+
+    @dtypes(*torch.testing.floating_and_complex_types())
+    def test_sparse_backward(self, device, dtype):
+        class FixedGradientFunction(Function):
+            @staticmethod
+            def forward(ctx, x, grad_x):
+                ctx.save_for_backward(grad_x)
+                return x
+
+            @staticmethod
+            def backward(ctx, grad_x):
+                saved_grad_x, = ctx.saved_tensors
+                return saved_grad_x, None
+
+        size = torch.Size([6, 3, 2])
+        i1 = torch.tensor([
+            [0, 3, 4],
+            [0, 2, 2],
+        ], dtype=torch.long)
+        v1 = torch.tensor([[1, 2], [4, 5], [7, 8]], dtype=dtype)
```

Comment:
use make_tensor(), this test currently ignores device, make_tensor() helps avoid that issue (while providing other features) by requiring device and dtype be specified